SAMtags.tex

```diff
@@ -213,8 +215,8 @@ \subsection{Additional Template and Mapping data}
 \emph{Strand} is either `{\tt +}' or `{\tt -}', indicating positive/negative strand, corresponding to FLAG bit 0x10.
 \emph{Pos} is a 1-based coordinate.
 
-\item[SM:i:\tagvalue{}]
-Template-independent mapping quality.
+\item[SM:i:\tagvalue{score}]
+Template-independent mapping quality, i.e., the mapping quality if the read were mapped as a single read rather than as part of a read pair or template.
 
 \item[TC:i:\tagvalue{}]
```

Comment:
`\tagvalue{number}`

Reply:
AS is already described as a _score_. This PR adds `\tagvalue{score}` to AM, MQ, PQ, and SM. I suppose AM/MQ/SM are properly mapping qualities, which (in bwa at least) are alignment scores scaled in some way, but therefore are not exactly scores themselves. And PQ is a Phred likelihood, with whatever relationship to alignment scores or mapping qualities that might have.

So perhaps `\tagvalue{mapq}` for AM/MQ/SM? Would rather not `number` as leaving it blank would convey the same amount of information…